### DISCUSSION TITLE
How to get rid of redundant hyphen chars when inserting space or tab?

### DISCUSSION THREAD

**Discussion by rvragul** [lukas-reineke/indent-blankline.nvim#594] [Q&A] [answered]
I made the following changes in config.

```lua
set.list = true
set.listchars:append "eol:↴"

require('indent_blankline').setup {
    char = '│',
    show_trailing_blankline_indent = false,
    show_end_of_line = true,
}

```

Now eol characters are inserted as expected. But hyphens are inserted when inserting space or tab chars.
I added a screenshot for reference.

![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/85612319/f5366487-bda4-45ff-9dbf-57f57b7812d7)

**Comment by lukas-reineke** [comment, marked as answer]
This is not coming from indent-blankline. That is the default Neovim value for trailing spaces. See `:help 'listchars'`
